DEVELOPER_GUIDE.md

```diff
@@ -100,7 +178,6 @@ Invoke integration tests by running the following command within the root direct
 make it
 ```
 
-Integration tests are expected to run for approximately 20-30 mins.
```

Comment:
Why was this line removed?  Is the duration different for WSL, perhaps?

Reply:
I placed it in the line above and bolded the timeframe there. I did this because I had not seen it the first time and I was confused as to why it was taking so long.